tests/ssl-opt.sh

```diff
@@ -156,6 +156,26 @@ requires_config_disabled() {
     fi
 }
 
+requires_config_value_at_least() {
+    NAME="$1"
+    DEF_VAL=$( grep ".*#define.*MBEDTLS_SSL_DTLS_MAX_BUFFERING" ../include/mbedtls/config.h |
```

Comment:
This computation can be extracted to a separate function so that it is not duplicated here and in requires_config_at_most.